docker-compose.yaml

```diff
@@ -49,7 +49,7 @@ services:
       STARTUP_COMMAND_1: ../wait-proto.sh
       STARTUP_COMMAND_2: ../npm-install.sh
       STARTUP_COMMAND_3: npm run typesafe-i18n
-      DEBUG: "socket,space"
+      DEBUG: "socket,space,matrix:*"
```

Comment:
The DEBUG environment variable is only used by the pusher.
On the front side, you need to put a DEBUG local storage variable to enable logs.